fv3gfs-physics/tests/savepoint/conftest.py

```diff
@@ -48,8 +57,7 @@ def data_path(pytestconfig):
 def data_path_from_config(config):
     data_path = config.getoption("data_path")
     namelist_filename = os.path.join(data_path, "input.nml")
-    fv3core._config.set_namelist(namelist_filename)
-    return data_path
+    return data_path, namelist_filename
```

Comment:
Please update the function name, and possibly also add a return value type hint.